src/extension.ts

```diff
@@ -29,23 +30,25 @@ export async function activate(context: ExtensionContext): Promise<void> {
     workspace.registerTextDocumentContentProvider(PrettyAnsiContentProvider.scheme, prettyAnsiContentProvider)
   );
 
-  const showPretty = async (options?: TextDocumentShowOptions) => {
-    const actualUri = window.activeTextEditor?.document.uri;
+  const showPretty = async (forcePreview: boolean = false, options: TextDocumentShowOptions = { viewColumn: ViewColumn.Active}) => {
```

Comment:
I am against positional boolean parameters, because they are difficult to understand at call site (i.e. what does `true` in `doThing(true)` even mean?)

I'd change this to something more like `showPretty({ viewColumn: ..., forcePreview: true })`.